django_markdown2/templatetags/md2.py

```diff
@@ -33,8 +37,10 @@ def markdown(value, arg=''):
         import markdown2
     except ImportError:
         if settings.DEBUG:
-            raise template.TemplateSyntaxError, "Error in {% markdown %} filter: The python-markdown2 library isn't installed."
-        return force_unicode(value)
+            raise template.TemplateSyntaxError("Error in {% markdown %} filter: The python-markdown2 library isn't installed.")
+        if sys.version_info.major == 2:
+            value = force_unicode(value)
```

Comment:
Please, use such code at the top of this file:
```
if sys.version_info.major == 2:
    from django.utils.encoding import force_unicode
else:
    force_unicode = lambda text: text
```
instead of this hack.